### DISCUSSION TITLE
How set different interval in watchtower for two different private repo images

### DISCUSSION THREAD

**Discussion by yuvrajalpha** [containrrr/watchtower#984] [General] [answered]
We have 10 containers running on Ubuntu device.
We have to monitor and update containers with different interval .

Can you please suggest the way to do it.

**Comment by simonaronsson** [comment, marked as answer]
Hi,

We currently don't support setting intervals per container. This feature request is tracked in https://github.com/containrrr/watchtower/issues/686.

Until then, you may accomplish this using multiple watchtower instances with different scopes. See https://containrrr.dev/watchtower/arguments/#filter_by_scope for more details.
